lib/rules/require-lang-attribute.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const Rule = require('./base');
+const AstNodeInfo = require('../helpers/ast-node-info');
+
+const ERROR_MESSAGE = 'The `<html>` element must have the `lang` attribute with a non-null value';
+
+module.exports = class RequireLangAttribute extends Rule {
+  visitor() {
+    return {
+      ElementNode(node) {
+        if (node.tag === 'html' && !AstNodeInfo.hasAttribute(node, 'lang')) {
+          this.log({
+            message: ERROR_MESSAGE,
+            line: node.loc && node.loc.start.line,
+            column: node.loc && node.loc.start.column,
+            source: this.sourceForNode(node),
+          });
+        }
+        const hasLangAttribute = AstNodeInfo.hasAttribute(node, 'lang');
+        const langAttributeValue = AstNodeInfo.elementAttributeValue(node, 'lang');
```

Comment:
`AstNodeInfo.elementAttributeValue` 😭 

- This will need to be added to #1388's TODOs